File: python/Mahasiswa.py
class Mahasiswa():
    __name = ""
    __NIM = ""
    __prodi = ""
    __fakultas = ""

    # constructor
    def __init__(self, name="", NIM="", prodi="", fakultas=""):
        self.__name = name
        self.__NIM = NIM
        self.__prodi = prodi
        self.__fakultas = fakultas

    # Setter methods
    def setName(self, name):
        self.__name = name

    def setNIM(self, NIM):
        self.__NIM = NIM

    def setProdi(self, prodi):
        self.__prodi = prodi

    def setFakultas(self, fakultas):
        self.__fakultas = fakultas

        # Getter methods
    def getName(self):
        return self.__name

    def getNIM(self):
        return self.__NIM

    def getProdi(self):
        return self.__prodi

    def getFakultas(self):
        return self.__fakultas

File: java/Mahasiswa.java
public class Mahasiswa {
    // name mhs
    private String name;
    // NIM mhs
    private String NIM;
    // prodi mhs
    private String prodi;
    // fakultas
    private String fakultas;

    Mahasiswa() {} // konstruktor Mahasiswa

    Mahasiswa(String name, String NIM, String prodi, String fakultas) {
        // konstruktor langsung isi attribut
        this.name = name;
        this.NIM = NIM;
        this.prodi = prodi;
        this.fakultas = fakultas;
    }

    /* Getter and setter */

    // setter
    public void setName(String nm) {
        this.name = nm;
    }

    public void setNIM(String nim) {
        this.NIM = nim;
    }

    public void setProdi(String prd) {
        this.prodi = prd;
    }

    public void setFakultas(String fklts) {
        this.fakultas = fklts;
    }

    // getter
    public String getName() {
        return name;
    }

    public String getNIM() {
        return NIM;
    }

    public String getProdi() {
        return prodi;
    }
    public String getFakultas() {
        return fakultas;
    }
    
}
